### PR TITLE
Ana/enable akash testnet

### PR DESCRIPTION
**Pull request by Bitcoinera**
Enable Akash testnet

### CODE REVIEW

**Comment by Bitcoinera** [comment]
Tested also with extension. Everything working great

**Comment by Bitcoinera** [comment]
There is no faucet yet it seems (the one [here ](https://faucet.akash.vitwit.com/)is for an old testnet), so ping me if you need any AKT

**Comment by faboweb** [comment]
🚀 Deployed on https://5ede96f1c623d7facea978e7--lunieio.netlify.app

File: app/changes/ana_enable-akash [hunk 1]
```diff
@@ -0,0 +1 @@
+[Changed] [#4223](https://github.com/cosmos/lunie/pull/4223) Enables Akash testnet @Bitcoinera
```
Comment (faboweb) [inline app/changes/ana_enable-akash:1]: wrong folder for the changelog

Comment (Bitcoinera) [reply app/changes/ana_enable-akash:1]: oops, let me fix it

**Pull request merged**